### PR TITLE
Add Technicals Interview

### PR DESCRIPTION
**Pull request by davidgasquez**
Everything you need to kick ass on your coding interview.

P.S: It's not a pure **awesome** repository, but I think it belongs here

### CODE REVIEW

**Comment by n1trux** [comment]
It doesn't have a license. Please read the [awesome manifesto](../blob/master/awesome.md).

Furthermore I don't find Quora links like "What is the difference between a process and a thread?" particularly awesome.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/blob/master/create-list.md

**Comment by davidgasquez** [comment]
I'll talk with @andreis to address those issues.

**Comment by andreis** [comment]
> It doesn't have a license

Laughs were had.

**Comment by maximveksler** [comment]
Super awesome. I love it!

**Comment by andreis** [comment]
https://github.com/andreis/interview/commit/a7f48fff9b65f41f3f82da2e8535c19710fee8be

I'm not sure whether there was a real need for a license, but I aim to please.

@n1trux I plan to add some improvements once the repo hits 2k stars, however the processes vs threads question comes up a lot in phone screens.

**Comment by sindresorhus** [comment]
@andreis WTFPL is not a real license. Please read: https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license

File: readme.md [hunk 1]
```diff
@@ -302,6 +302,7 @@ If you want to create your own list, please read the [create list instructions](
 - [GraphQL](https://github.com/chentsulin/awesome-graphql)
 - [Transit](https://github.com/luqmaan/awesome-transit)
 - [Research Tools](https://github.com/emptymalei/awesome-research)
+- [Coding/Technicals Interview](https://github.com/andreis/interview)
```
Comment (sindresorhus) [inline readme.md]: Should be `Coding Interview`.

The repo name is also a bit ambiguous. `interview` doesn't say much.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**